app/services/otp_rate_limiter.rb

```diff
@@ -35,6 +38,15 @@ def lock_out_user
   def increment
     # DO NOT MEMOIZE
     @entry = OtpRequestsTracker.atomic_increment(entry_for_current_phone.id)
+    throttle.increment!
+  end
```

Comment:
the return value of these is going to differ. should we just have it return an explicit `nil` to make sure we don't use the results accidentally?

```suggestion
  def increment
    # DO NOT MEMOIZE
    @entry = OtpRequestsTracker.atomic_increment(entry_for_current_phone.id)
    throttle.increment!
    nil
  end
```